libs/sdk-core/src/lib/rest/rest.ts

```diff
@@ -86,24 +85,16 @@ export class Rest implements IRest {
     synchronous = false,
   ): Observable<T> {
     const specialContentType = extraHeaders && extraHeaders['content-type'];
-    return fromFetch(this.getFullUrl(path), {
-      selector: (response) => Promise.resolve(response),
-      headers: this.getHeaders(method, path, extraHeaders, synchronous),
-      method,
-      body: specialContentType ? body : JSON.stringify(body),
-    }).pipe(
-      switchMap((response) => {
-        if (!response.ok) {
-          return throwError(() => response);
+    return from(
+      fetch(this.getFullUrl(path), {
+        method,
+        headers: this.getHeaders(method, path, extraHeaders, synchronous),
+        body: specialContentType ? body : JSON.stringify(body),
+      }).then((res) => {
+        if (!res.ok) {
+          throw new Error(`${res.status}`);
```

Comment:
why do you return only the status in the Error? I a lot of cases we don't use error details, but there are some cases (like in the manager) where having the details would be valuable. I feel like here you're removing too much informations from the error and I don't really understand what's the point, why not throwing the actual error (and so `res` entirely I guess).

Reply:
As the backend uses a status for a unique reason on each endpoint, we do not loose any information by just keeping the status.

Moreover, error management was really hell in the desktop app and, psychologically, I am not ready to invest time to improve it now.

So let's address that in a future task if at some point we think having more detail errors is useful.